components/automate-ui/src/app/app-routing.module.ts

```diff
@@ -164,7 +164,8 @@ const routes: Routes = [
         },
         {
           path: 'node-credentials',
-          loadChildren: './pages/+compliance/+credentials/credentials.module#CredentialsModule'
+          loadChildren: () => import('./pages/+compliance/+credentials/credentials.module')
+            .then(m => m.CredentialsModule)
```

Comment:
This change was an auto-fix from the Angular migration; you will see this repeated many times.